src/styles/less/ParentFooter.less

```diff
@@ -0,0 +1,43 @@
+/* Footer Styles */
+
+.no-scroll {
+	position: absolute;
+	bottom: 0;
+	width: 100%;
+}
+
+footer {
+	// position: absolute;
```

Comment:
Great job catching the bug here.  Absolute renders differently depending on how each component is styled.